<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>725</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>725</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="tv_sprcontent">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>256</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>384</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="autoScroll">
        <bool>false</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="tabKeyNavigation">
        <bool>true</bool>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="uniformRowHeights">
        <bool>false</bool>
       </property>
       <property name="animated">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
      <widget class="QStackedWidget" name="stkEditor">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>760</width>
         <height>540</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="currentIndex">
        <number>7</number>
       </property>
       <widget class="TabEmpty" name="tabEmpty">
        <property name="minimumSize">
         <size>
          <width>760</width>
          <height>540</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
       <widget class="TabWelcome" name="tabWelcome"/>
       <widget class="TabProperties" name="tabProperties"/>
       <widget class="TabImages" name="tabImages"/>
       <widget class="TabFrames" name="tabFrame"/>
       <widget class="TabAnimSequence" name="tabSequence">
        <property name="minimumSize">
         <size>
          <width>760</width>
          <height>540</height>
         </size>
        </property>
       </widget>
       <widget class="TabUnknown" name="tabUnknown"/>
       <widget class="TabAnimTable" name="tabAnimTable">
        <property name="minimumSize">
         <size>
          <width>760</width>
          <height>540</height>
         </size>
        </property>
       </widget>
       <widget class="TabAnimGroup" name="tabAnimGroup"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menu_New">
     <property name="title">
      <string>&amp;New</string>
     </property>
     <addaction name="actionNewSprite"/>
     <addaction name="actionNewSprite_Pack_File"/>
     <addaction name="separator"/>
     <addaction name="actionAdvanced"/>
    </widget>
    <addaction name="menu_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Recent"/>
    <addaction name="separator"/>
    <addaction name="action_Save"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="action_Export"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="action_Settings"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="actionShow_Start_Screen"/>
    <addaction name="actionReport_a_bug"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionSprite_Sheet_Auto_Importer"/>
    <addaction name="actionSprite_Validator"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Tools"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::NoToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="actionSave_As"/>
   <addaction name="separator"/>
   <addaction name="action_Export"/>
   <addaction name="actionSprite_Sheet_Auto_Importer"/>
   <addaction name="separator"/>
   <addaction name="action_AddSprite"/>
   <addaction name="action_RemoveSprite"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="../../appres.qrc">
     <normaloff>:/resources/fugue-icons-3.5.6/icons/disk.png</normaloff>:/resources/fugue-icons-3.5.6/icons/disk.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="../../appres.qrc">
     <normaloff>:/resources/fugue-icons-3.5.6/icons/disk--arrow.png</normaloff>:/resources/fugue-icons-3.5.6/icons/disk--arrow.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As..</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_Export">
   <property name="icon">
    <iconset resource="../../appres.qrc">
     <normaloff>:/resources/fugue-icons-3.5.6/icons/document-export.png</normaloff>:/resources/fugue-icons-3.5.6/icons/document-export.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Export..</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Allow exporting the current sprite's content, or the current item to the desired format.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="icon">
    <iconset resource="../../appres.qrc">
     <normaloff>:/resources/fugue-icons-3.5.6/icons/door--arrow.png</normaloff>:/resources/fugue-icons-3.5.6/icons/door--arrow.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionNewSprite">
   <property name="icon">
    <iconset resource="../../appres.qrc">
     <normaloff>:/resources/fugue-icons-3.5.6/icons/blue-document-medium.png</normaloff>:/resources/fugue-icons-3.5.6/icons/blue-document-medium.png</iconset>
   </property>
   <property name="text">
    <string>Sprite..</string>
   </property>
   <property name="toolTip">
    <string>Create a new sprite</string>
   </property>
  </action>
  <action name="actionNewSprite_Pack_File">
   <property name="icon">
    <iconset resource="../../appres.qrc">
     <normaloff>:/resources/fugue-icons-3.5.6/icons/box-medium.png</normaloff>:/resources/fugue-icons-3.5.6/icons/box-medium.png</iconset>
   </property>
   <property name="text">
    <string>Sprite Pack File..</string>
   </property>
   <property name="toolTip">
    <string>Create a new sprite database file.</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="../../appres.qrc">
     <normaloff>:/resources/fugue-icons-3.5.6/icons/arrow-curve-180-left.png</normaloff>:/resources/fugue-icons-3.5.6/icons/arrow-curve-180-left.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../appres.qrc">
     <normaloff>:/resources/fugue-icons-3.5.6/icons/arrow-curve.png</normaloff>:/resources/fugue-icons-3.5.6/icons/arrow-curve.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="action_Settings">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../appres.qrc">
     <normaloff>:/resources/fugue-icons-3.5.6/icons/gear--pencil.png</normaloff>:/resources/fugue-icons-3.5.6/icons/gear--pencil.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Settings..</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="../../appres.qrc">
     <normaloff>:/resources/fugue-icons-3.5.6/icons/information-white.png</normaloff>:/resources/fugue-icons-3.5.6/icons/information-white.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About..</string>
   </property>
  </action>
  <action name="action_Recent">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Recent</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="../../appres.qrc">
     <normaloff>:/resources/fugue-icons-3.5.6/icons/folder-open-document.png</normaloff>:/resources/fugue-icons-3.5.6/icons/folder-open-document.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open..</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Open a sprite file, or a sprite container for editing!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionShow_Start_Screen">
   <property name="text">
    <string>Show Start Screen </string>
   </property>
  </action>
  <action name="actionSprite_Sheet_Auto_Importer">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../appres.qrc">
     <normaloff>:/resources/fugue-icons-3.5.6/icons/slide--arrow.png</normaloff>:/resources/fugue-icons-3.5.6/icons/slide--arrow.png</iconset>
   </property>
   <property name="text">
    <string>Sprite Sheet Auto Importer..</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tool to import an entire sprite sheet and automatically generate animation sequences, images, and frames from it. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionSprite_Validator">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../appres.qrc">
     <normaloff>:/resources/fugue-icons-3.5.6/icons/magnifier--exclamation.png</normaloff>:/resources/fugue-icons-3.5.6/icons/magnifier--exclamation.png</iconset>
   </property>
   <property name="text">
    <string>Sprite  Validator..</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;A tool meant to check for common errors and quickly fix them, to save the user  some time!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="action_AddSprite">
   <property name="icon">
    <iconset resource="../../appres.qrc">
     <normaloff>:/resources/fugue-icons-3.5.6/icons/blue-document--plus.png</normaloff>:/resources/fugue-icons-3.5.6/icons/blue-document--plus.png</iconset>
   </property>
   <property name="text">
    <string>Add Sprite</string>
   </property>
  </action>
  <action name="action_RemoveSprite">
   <property name="icon">
    <iconset resource="../../appres.qrc">
     <normaloff>:/resources/fugue-icons-3.5.6/icons/blue-document--minus.png</normaloff>:/resources/fugue-icons-3.5.6/icons/blue-document--minus.png</iconset>
   </property>
   <property name="text">
    <string>Remove Sprite</string>
   </property>
   <property name="toolTip">
    <string>Remove Sprite</string>
   </property>
  </action>
  <action name="actionReport_a_bug">
   <property name="icon">
    <iconset resource="../../appres.qrc">
     <normaloff>:/resources/fugue-icons-3.5.6/icons/bug--arrow.png</normaloff>:/resources/fugue-icons-3.5.6/icons/bug--arrow.png</iconset>
   </property>
   <property name="text">
    <string>Report a bug</string>
   </property>
  </action>
  <action name="actionAdvanced">
   <property name="text">
    <string>Adva&amp;nced..</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Create a new item to work on and select precisely what you need..&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TabAnimSequence</class>
   <extends>QWidget</extends>
   <header location="global">src/ui/tabs/tabanimsequence.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabAnimTable</class>
   <extends>QWidget</extends>
   <header location="global">src/ui/tabs/tabanimtable.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabProperties</class>
   <extends>QWidget</extends>
   <header location="global">src/ui/tabs/tabproperties.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabImages</class>
   <extends>QWidget</extends>
   <header location="global">src/ui/tabs/tabimages.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabFrames</class>
   <extends>QWidget</extends>
   <header location="global">src/ui/tabs/tabframes.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabEmpty</class>
   <extends>QWidget</extends>
   <header location="global">src/ui/tabs/tabempty.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabWelcome</class>
   <extends>QWidget</extends>
   <header location="global">src/ui/tabs/tabwelcome.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabAnimGroup</class>
   <extends>QWidget</extends>
   <header location="global">src/ui/tabs/tabanimgroup.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabUnknown</class>
   <extends>QWidget</extends>
   <header location="global">src/ui/tabs/tab_unknown.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../appres.qrc"/>
 </resources>
 <connections/>
</ui>
